packages/core/src/base-axis-chart.ts

```diff
@@ -12,6 +12,10 @@ import { BaseChart } from "./base-chart";
 import * as Configuration from "./configuration";
 import { Tools } from "./tools";
 
+import { zoom } from "d3-zoom";
+import { drag } from "d3-drag";
+import { marcon } from "d3-marcon"
```

Comment:
Also, the `d3-marcon` package doesn't seem to be a reputable package, or part of the `d3` community. Do we really need to use it?